src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeed.cs

```diff
@@ -22,25 +22,35 @@ namespace Microsoft.DotNet.Build.Tasks.Feed
     public sealed class BlobFeed
     {
         private MSBuild.TaskLoggingHelper Log;
+
         public string AccountName { get; set; }
+
         public string AccountKey { get; set; }
+
         public string ContainerName { get; set; }
+
+        public string RelativePath { get; set; }
+
+        public string SecurePath { get; set; }
```

Comment:
Lets not even call this secure path. We should just call it FeedContainerUrl like you do in the code.

Reply:
Also it doesn't look like this is the full url like we talked.

Reply:
Sorry you did the the Url in the outter task, however I think you should be passing the full URL to this class instead of reconstructing it as that defeats the part of the purpose.